### PR TITLE
feat: 🎸 add form page with recap header. needs form component

### LINKED ISSUE #117: [Layout] Recap of what the Resilience is offering
Once the user is directed to the Get Resilience form, there will be a quick recap of all the ways that Resilience would be beneficial to organizations at the top of the page.

**Description:**
On the left side with will be a sales pitch - Left Aligned
A link to display how our users use the application - Left Aligned
The right side will be a checklist of features stored in a box - Center Aligned

[Mockup](https://www.figma.com/file/GLg0zPNnwMEGIb9TytCrxQ/Resilience-App?node-id=33854%3A19521)

**Screenshot:**

### PR DESCRIPTION
**Pull request by jessdwitch**
Title:
Add recap of what Resilience is offering

Description:
This closes #117 and #116  (sorry they seem to be dependent on each other since this adds a field to the graphql query and flips out if the field isn't defined somewhere like the .md file for 116)
It's a template page for the form-page that includes the recap header, but is missing the form itself (it looks like there's another PR with that up).

Both features and content provided:
image

Content only:
image

Features only:
image

Neither:
image

Smooshed:
image

### CODE REVIEW

File: src/templates/form-page.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+      </div>
+    </div>
+  );
+};
+
+FormPageTemplate.propTypes = {
+  title: PropTypes.string.isRequired,
+  content: PropTypes.string,
+  contentComponent: PropTypes.func,
+  features: PropTypes.arrayOf(PropTypes.string),
```
Comment (jwu910) [inline src/templates/form-page.js:38]: Do we need to add any type of existing check to hide/show these elements in the render if optional props do not exist?

example: If `content` is undefined, buy `contentComponent` is provided, will `undefined` get passed into the content prop?

Comment (jessdwitch) [reply src/templates/form-page.js:38]: I was thinking that too, and decided against adding it, just cause the PR was getting big, and it's not strictly in the design (figured I'd do a follow up PR without an issue number). I have the code to collapse the header without content, and the feature list if that list is empty in an experimental branch I can clean up and throw in later tonight.

**Comment by jessdwitch** [comment]
Coming back to this in a day or 2. Ended up going through the weekend on some work junk.

File: src/templates/form-page.js (hunk 1 above)
Comment (jessdwitch) [reply src/templates/form-page.js:64]: New commit adds conditional render for optionals. Screenshots in the description 😄

**Comment by jessdwitch** [comment]
https://deploy-preview-133--resilience-landing-page.netlify.app/get-resilience/
will get you straight there since the Get Resilience button isn't wired up.

> Deploy preview for _resilience-landing-page_ ready!
> 
> Built with commit [cd722bd](https://github.com/factn/resilience-landing-page/commit/cd722bd86ff4ed067370c51cda6d901d4cf0f676)
> 
> https://deploy-preview-133--resilience-landing-page.netlify.app

File: src/components/scss/_featureList.scss [hunk 2]
```diff
@@ -0,0 +1,31 @@
+.featureList_container {
+  border: $color-grey;
+  border-style: solid;
+  border-width: 1px;
```
Comment (jwu910) [inline src/components/scss/_featureList.scss:4]: FYI these 3 can be shorthanded into `border: 1px solid $color-grey`

**Comment by jwu910** [comment]
Cool, just got a minute to poke around the `/get-resilience` route. Looks great! Thanks @jessdwitch 
I think we need a ticket (if we don't have one already) to wire up the button on the nav to this page. and then load the component in, right?

Apologies for being disconnected with the next steps, @DAHuerta could you take a look when you get a minute too?

**Comment by jwu910** [comment]
@all-contributors please add @jessdwitch for code

**Pull request merged**